package.json

```diff
@@ -41,6 +41,7 @@
     "gts": "^0.9.0",
     "jest": "^24.8.0",
     "jest-date-mock": "^1.0.7",
+    "pretty-format": "^24.9.0",
```

Comment:
This is a jest dependency, but we're using it explicitly in the custom cmp expectations.